google/cloud/spanner/sql_partition_test.cc

```diff
@@ -19,85 +19,89 @@ namespace google {
 namespace cloud {
 namespace spanner {
 inline namespace SPANNER_CLIENT_NS {
+
 class SqlPartitionTester {
  public:
   SqlPartitionTester() = default;
   SqlPartitionTester(SqlPartition partition)
       : partition_(std::move(partition)) {}
-  SqlStatement const& sql_statement() const {
-    return partition_.sql_statement();
-  }
-  std::string const& partition_token() const {
+  SqlStatement const& Statement() const { return partition_.sql_statement(); }
```

Comment:
Unless clang-tidy complained, I think the lower-case spelling for these getters is preferable.

https://google.github.io/styleguide/cppguide.html#Function_Names

"Accessors and mutators (get and set functions) may be named like variables. These often correspond to actual member variables, but this is not required."

Reply:
clang-tidy complained. I had originally named them in the accessor style.

Reply:
You can silence clang-tidy with `// NOLINT` or with `// NOLINTNEXTLINE`.... the robots are here to help us, but we still have control of the power cord :grin: 

#include <coryan-rants-about-google-style-guide>

Reply:
Point taken. Next time I'm in this file, I'll do the needful.